### DISCUSSION TITLE
🔥 FrontendBR Space • Edição Front in Campinas + Cupom de desconto

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#1384] [Discussão]
![image](https://user-images.githubusercontent.com/3603793/54364802-5e982980-464c-11e9-84ba-ff5e7cf41f3e.png)

Depois do sucesso durante o Front in Sampa, é com muito orgulho que oficializamos mais um **FrontendBR Space**! 

Dessa vez durante o [Front in Campinas](https://frontincampinas.com.br/) que acontece no dia 04/Maio/2019 😄 

**FrontendBR Space** é um espaço aberto e pensado para todos vocês relaxarem e se divertir durante o evento. Temos um cupom de desconto no valor de 10% para o evento:

```
frontendbr
```

A primeira edição aconteceu no Front in Sampa e podem ver mais informações na issue #1124. Nos próximos dias vamos confirmar todas as atrações e atividades que vão acontecer no espaço.

Esse é um presente do Front in Campinas e do Front-end BR  ❤️

**Comment by notnotgabriel** [comment]
uhulllllllll 👯 
só venham :)

**Comment by Gpaula95** [comment]
No Front in Vale tem que ter também ein ! @felipefialho

**Comment by diogomoretti** [comment]
Bóóóóóóóraaa!!! 😍

**Comment by hugooliveirad** [comment]
Boa!

**Comment by klimbo123** [comment]
boaa
[Bluestacks](https://bluestacks.vip/) [Lucky Patcher](https://luckypatcher.pro/) [Kodi](https://kodi.software/)
